webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/RoleEntity.java

```diff
@@ -8,20 +8,20 @@
 package io.camunda.webapps.schema.entities.usermanagement;
 
 import io.camunda.webapps.schema.entities.AbstractExporterEntity;
-import java.util.Set;
 
 public class RoleEntity extends AbstractExporterEntity<RoleEntity> {
 
-  private Long roleKey;
+  private Long key;
```

Comment:
🙃 Just a small nitpick, we can keep the `roleKey`change  as we have a `memberKey` as well now.

Reply:
I've changed it to match the accessor names and we can't change those as the convention is to have a `"key"` field in ES.